cpu/stm32f1/lpm_arch.c

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright (C) 2013 INRIA
- * Copyright (C) 2014 Freie Universität Berlin
```

Comment:
I meant that the file is a copy-paste of the one for stm32f2, the author of which is @DipSwitch not myself. I just copied over. Hence I left the author and copyright as in the original file. I can add myself but as I said, I didn't write a single line of code on this file :-)